test/torchaudio_unittest/prototype/ctc_decoder_test.py

```diff
@@ -64,6 +64,16 @@ def test_shape(self):
         results = decoder(emissions)
         self.assertEqual(len(results), emissions.shape[0])
 
+    def test_timesteps_shape(self):
+        """Each token should correspond with a timestep"""
+        emissions = self._get_emissions()
+        decoder = self._get_decoder()
+
+        results = decoder(emissions)
+        for i in range(emissions.shape[0]):
+            result = results[i][0]
+            self.assertEqual(result.tokens.shape[0], result.timesteps.shape[0])
```

Comment:
Can we compare the whole shape?
```suggestion
            self.assertEqual(result.tokens.shape, result.timesteps.shape)
```